src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/CommandBar/BackButtonImage/CommandBar_Page2.xaml.cs

```diff
@@ -39,9 +39,12 @@ public void OnButtonClicked(object sender, object args)
 
 	        var navigationBar = parent.FindFirstChild<UnoNavigationBar>();
 
-	        var image = navigationBar.BackIndicatorImage;
-	        ExpectedImage.Source = image;
+			var image = (UIDevice.CurrentDevice.CheckSystemVersion(13, 0))
+				? navigationBar.CompactAppearance.BackIndicatorImage
```

Comment:
Confused why we are looking at the Compact Appearance here. It should definitely be working to have a custom view for the back button